src/pyOpenMS/pxds/NLargest.pxd

```diff
@@ -10,10 +10,12 @@ cdef extern from "<OpenMS/FILTERING/TRANSFORMERS/NLargest.h>" namespace "OpenMS"
     cdef cppclass NLargest(DefaultParamHandler):
         # wrap-inherits:
         #    DefaultParamHandler
-
-        NLargest() nogil except + # wrap-doc:NLargest removes all but the n largest peaks
+        # wrap-doc:
+        #   NLargest removes all but the n largest peaks
+        
+        NLargest() nogil except + 
         NLargest(NLargest &) nogil except +
 
-        void filterSpectrum(MSSpectrum & spec) nogil except +
-        void filterPeakSpectrum(MSSpectrum & spec) nogil except +
-        void filterPeakMap(MSExperiment & exp) nogil except +
+        void filterSpectrum(MSSpectrum & spec) nogil except + # wrap-doc:Removes all except spectrums
+        void filterPeakSpectrum(MSSpectrum & spec) nogil except + # wrap-doc:Removes all except peak spectrums
+        void filterPeakMap(MSExperiment & exp) nogil except + # wrap-doc:Removes all except peak map
```

Comment:
I don't get the documentation here.

Reply:
I took reference from the class definition ( because I didn't find anything about the function in the header file ) which was **NLargest removes all but the n largest peaks**.
Any suggestions?

Reply:
I see. The documentation in the header is indeed missing.
It should be something like:
# wrap-doc:Keep only n-largest peaks in spectrum
# wrap-doc:Keep only n-largest peaks in spectrum
# wrap-doc:Keep only n-largest peaks in each spectrum of a peak map

Reply:
@jpfeuffer do you know why we would need both?